test/test_per_overload_api.py

```diff
@@ -60,5 +60,9 @@ def test_basics_opoverload(self):
 
         self.assertRaises(RuntimeError, lambda: add_tensoroverload(a, a, out=b))
 
+    def test_dir(self):
+        for n in dir(torch.ops.aten):
+            self.assertTrue(hasattr(torch.ops.aten, n), n)
```

Comment:
Should we check that `len(dir(torch.ops.aten)) > SOME_NUMBER`? I'd expect this test to pass before this PR